frontend/packages/console-dynamic-plugin-sdk/src/api/api.ts

```diff
@@ -1,11 +1,18 @@
-import { UseK8sWatchResource, UseK8sWatchResources, UseResolvedExtensions } from './api-types';
+import { MockImpl, MockCoFetchJSONImpl } from './api-mocks';
+import {
+  UseK8sWatchResource,
+  UseK8sWatchResources,
+  UseResolvedExtensions,
+  CoFetch,
+  CoFetchJSON,
+  CoFetchText,
+} from './api-types';
 
 export * from './api-types';
 
-const MockImpl = () => {
-  throw new Error('You need to configure webpack externals to use this function at runtime.');
-};
-
 export const useK8sWatchResource: UseK8sWatchResource = MockImpl;
```

Comment:
```ts
export const coFetch: CoFetch = newMockImpl();
```

This pattern can be used consistently throughout the API code.

I think it's better for each exposed API object to have its own mock instance.

Reply:
Updated.